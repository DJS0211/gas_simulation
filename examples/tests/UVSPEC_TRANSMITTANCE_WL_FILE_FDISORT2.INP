data_files_path ../../data/
atmosphere_file midlatitude_summer
source solar ../../data/solar_flux/kurudz_1.0nm.dat

wavelength 1064
mol_tau_file abs ../UVSPEC_TRANSMITTANCE_MOL_TAU_FILE_AFGLMS.09000.3wvn.cdf
wavelength_grid_file ../UVSPEC_TRANSMITTANCE_WL.DAT

rte_solver fdisort2        # Radiative transfer equation solver
quiet
