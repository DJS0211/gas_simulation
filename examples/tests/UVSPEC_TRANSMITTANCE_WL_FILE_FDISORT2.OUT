 1064.000  6.412084e+02  2.221136e+00  4.152471e-06  5.102575e+01  5.439398e-01  2.472698e-06 
